R/lockfile.R

```diff
@@ -5,7 +5,9 @@ renv_lockfile_init <- function(project) {
 
   lockfile$R        <- renv_lockfile_init_r(project)
   lockfile$Python   <- renv_lockfile_init_python(project)
-  lockfile$Packages <- list()
+  lockfile$quarto   <- renv_lockfile_quarto()
```

Comment:
I feel like we should collect these into a common lockfile entry, e.g. `lockfile$External$quarto` or something like that.